### DISCUSSION TITLE
私聊引用回复的问题

### DISCUSSION THREAD

**Discussion by heddxh** [Mrs4s/go-cqhttp#1278] [Q&A] [answered]
看到[这个issue下面的回复](https://github.com/Mrs4s/go-cqhttp/issues/1052#issuecomment-984713574) ，即私聊下完全没有解析引用回复。任何自己试了一下复现了。请问是还没支持还是bug？

**Comment by wdvxdr1123** [comment, marked as answer]
请尝试使用 https://github.com/Mrs4s/go-cqhttp/actions/runs/1598782096
